plugins/candela/web_client/templates/configView.pug

```diff
@@ -6,5 +6,7 @@ p
   |  to render table files directly in Girder.
 
 p.
-  This plugin requires no configuration. Simply upload a CSV file as an item,
-  then set the Candela visualization type and options.
+  This plugin requires no configuration. Simply upload a CSV file as an item
```

Comment:
I agree with the proposed changes and will rebase if the other PR is merged first.